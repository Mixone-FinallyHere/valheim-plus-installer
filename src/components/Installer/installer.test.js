import React from "react";
import Installer from ".";
import mock_state from "test_helpers/mock_state";
import { render } from "@testing-library/react";

it("renders without crashing", () => {
  const { asFragment } = render(
    <Installer state={mock_state} stateDispatch={jest.fn()} confirmInstallation={jest.fn()} />
  );

  expect(asFragment()).toMatchSnapshot();
});
