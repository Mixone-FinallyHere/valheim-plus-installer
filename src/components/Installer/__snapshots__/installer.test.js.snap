// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-2 makeStyles-logWindow-1"
  >
    <div
      class="MuiBox-root MuiBox-root-3"
    >
      Install Log
    </div>
    <ul
      class="MuiList-root MuiList-padding"
    >
      <li
        class="MuiListItem-root MuiListItem-gutters"
      >
        <div
          class="MuiListItemText-root"
        >
          <span
            class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
          >
            Welcome to the Valheim+ Installer
          </span>
        </div>
      </li>
    </ul>
    <div
      class="MuiBox-root MuiBox-root-4"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          OK
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</DocumentFragment>
`;
